espnet/bin/asr_recog.py

```diff
@@ -71,6 +73,13 @@ def get_parser():
                         help='Input length ratio to obtain min output length')
     parser.add_argument('--ctc-weight', type=float, default=0.0,
                         help='CTC weight in joint decoding')
+    # transducer related
+    parser.add_argument('--search-type', type=str, default='beam',
+                        choices=['greedy', 'beam'],
+                        help='Search algorithm to use for transducer.')
+    parser.add_argument('--score-norm', type=strtobool, nargs='?',
+                        default=True,
+                        help='Normalize transducer scores by length')
```

Comment:
Maybe `--score-norm-transducer` is better because this option can be seen from non-transducer models.